travis_ci_flake8.cfg

```diff
@@ -133,3 +129,15 @@ ignore =
     W601,
     # W605: invalid escape sequence '\d'
     W605,
+per-file-ignores =
+    src/bika/lims/skins/bika/guard_handler.py:F401,
+    src/bika/lims/browser/worksheet/views/__init__.py:F401,
+    src/bika/lims/browser/analyses/__init__.py:F401,
+    src/bika/lims/browser/client/__init__.py:F401,
+    src/bika/lims/browser/widgets/__init__.py:F401,
+    src/bika/lims/browser/analysisrequest/analysisrequests.py:F401,
+    src/bika/lims/browser/analysisrequest/__init__.py:F401,
+    src/bika/lims/subscribers/__init__.py:F401,
+    src/bika/lims/interfaces/__init__.py:F401,
+    src/senaite/core/exportimport/instruments/__init__.py:F401,
+    src/senaite/core/behaviors/__init__.py:F401,
```

Comment:
ah, got it, here you ignore them per file.
However, probably the local editor will still complain, but ok for now.

Reply:
>  However, probably the local editor will still complain, but ok for now.

good point :+1: maybe we can rename the config to `.flake8` after cleaning up and hopefully the editor(s) will recognize it?